doozer/doozerlib/distgit.py

```diff
@@ -1722,12 +1722,12 @@ def _update_image_config(self):
         """
 
         # Check if there is an alternative configuration matching upstream RHEL version
-        alt_configs = self.config.get('alternative_upstream', [])
+        alt_configs = self.config.alternative_upstream
         matched = False
         for alt_config in alt_configs:
```

Comment:
this will break if it's not there, can't iterate MissingModel
```suggestion
        for alt_config in alt_configs or []:
```